stable/kasm/1.12.0/README.md

```diff
@@ -0,0 +1,37 @@
+# Introduction
+
+Kasm Workspaces streaming containerized apps and desktops to end-users.
+
+TrueCharts are designed to be installed as TrueNAS SCALE app only. We can not guarantee this charts works as a stand-alone helm installation.
+**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/truecharts/apps/issues/new/choose)**
+
+## Source Code
+
+* <https://github.com/thelamer/charts-test/tree/main>
```

Comment:
do we want to be pointing at your repo, or the kasmtech repo?

Reply:
No we sure don't, updated both of them.